emission/analysis/classification/inference/mode.py

```diff
@@ -28,14 +35,11 @@ def __init__(self):
                           "start hour", "end hour", "close to bus stop", "close to train stop",
                           "close to airport"]
     self.Sections = edb.get_section_db()
-    self.PredictionDB = edb.get_prediction_db()
 
-  def runPipelineModelStage(self):
+  def runPipelineModelStage(self, uuid, timerange):
```

Comment:
so just to clarify, does this build the model and use it to generate predictions?
I thought we were going to split that...